$NetBSD: patch-doc_ssl_SSL__do__handshake.pod,v 1.1.6.1 2014/01/10 18:00:40 spz Exp $

Fix openssl pod docs to work with the very picky pod2man from perl-5.18.0.

--- doc/ssl/SSL_do_handshake.pod.orig	2014-01-06 13:47:42.000000000 +0000
+++ doc/ssl/SSL_do_handshake.pod	2014-01-10 13:32:08.000000000 +0000
@@ -45,13 +45,13 @@
 
 =over 4
 
-=item 0
+=item Z<>0
 
 The TLS/SSL handshake was not successful but was shut down controlled and
 by the specifications of the TLS/SSL protocol. Call SSL_get_error() with the
 return value B<ret> to find out the reason.
 
-=item 1
+=item Z<>1
 
 The TLS/SSL handshake was successfully completed, a TLS/SSL connection has been
 established.
